### DISCUSSION TITLE
how to drag window

### DISCUSSION THREAD

**Discussion by nyrf** [kovidgoyal/kitty#4248] [Q&A] [answered]
When i set `hide_window_decorations titlebar-only` on my macOS 10.14.6, i found can't drag window. i also try to  set `tab_bar_margin_height   10 0`

https://user-images.githubusercontent.com/299532/142202775-190d80cd-67f1-460a-bec3-1731fd932775.mp4

**Comment by kovidgoyal** [comment]
yes if you want to drag windows, you need window decorations, the two go
hand in hand

**Reply by nyrf** [reply]
> yes if you want to drag windows, you need window decorations, the two go hand in hand

Thanks,  i see.

**Comment by Luflosi** [comment, marked as answer]
You actually can drag the window, see https://github.com/kovidgoyal/kitty/issues/84#issuecomment-768214513 and #3683.
I created a first draft of a PR that improves the situation (#3744) but I haven't found the time and motivation to finish it.

**Reply by nyrf** [reply]
> You actually can drag the window, see [#84 (comment)](https://github.com/kovidgoyal/kitty/issues/84#issuecomment-768214513) and #3683.
> I created a first draft of a PR that improves the situation (#3744) but I haven't found the time and motivation to finish it.

Thanks, i've found the issues https://github.com/kovidgoyal/kitty/issues/3683#issuecomment-852922435, so I thought window could be dragged.

**Comment by mendhak** [comment]
Future searchers (as this is a top result, pardon) - on Linux you can use alt click to drag the window around, alt right click to resize the window. Hope this helps.

**Reply by xPlay5r** [reply]
If it doesnt work, try Window key + left and right click.

Depending on environment, these keybinds might not work
